examples/qualcomm/llama2/runner/runner.cpp

```diff
@@ -0,0 +1,386 @@
+/*
```

Comment:
curious if we can share this file with `models/llama2`? Just easier to maintain.